proj2/WiedzminGigabajt.md

```diff
@@ -0,0 +1,66 @@
+# Wiedźmin Gigabajt
+
+## Punktacja
+8 punktów za wersję podstawową
+
+po 6 punktów dla dwóch osób za wersję podstawową
+
+po 8 punktów dla dwóch osób za wersję z przynajmniej jednym rozszerzeniem
```

Comment:
Nie wiem czy nie wypada dać więcej punktów.

```suggestion
- 8 punktów za wersję podstawową

- po 6 punktów dla dwóch osób za wersję podstawową

- po 8 punktów dla dwóch osób za wersję z przynajmniej jednym rozszerzeniem
```

Reply:
[Według rozpiski na stronie głównej GH z PO](https://github.com/apohllo/obiektowe-lab), 8 pkt to maksymalna liczba możliwych punktów za drugi projekt.

Reply:
Dzień dobry, pomysł oczywiście wygląda sympatycznie (choć wymagania techniczne bardziej przypominają projekt 3-osobowy na TO na 3-cim roku :D). Natomiast żeby być uczciwym to powinien on przejść przez konkurs tematów, każdy prowadzący robił/będzie robił taką zabawę na swoich zajęciach. ;) I dopiero te wyłonione tematy trafią tutaj do puli.

Reply:
Dzień dobry, dziękuję za odpowiedź i wyjaśnienie szczegółów. Porozmawiam z swoim prowadzącym podczas zajęć. IMO projekt nie jest taki trudny oraz jest spokojnie wykonywalny przez jedną osobę :)

Pozdrawiam
Wojciech Barczyński